Mathlib/Algebra/Order/Pointwise.lean

```diff
@@ -9,7 +9,9 @@ Authors: Alex J. Best, Yaël Dillies
 ! if you have ported upstream changes.
 -/
 import Mathlib.Algebra.Bounds
-import Mathlib.Data.Set.Pointwise.Smul
+import Mathlib.Algebra.Order.Field.Basic
```

Comment:
Why do we need the new imports?

Reply:
Why do we need the new import?

Reply:
If you mean `Mathlib.Data.Set.Pointwise.SMul`, that is required for, for example, `mem_smul_set`.

Reply:
And `Mathlib.Algebra.Order.Field.Basic` is for instance `LinearOrderedField`.

Reply:
Thanks for all those help, I'll be more careful afterwards.